modules/_labs/pagination/react/README.md

```diff
@@ -14,82 +74,555 @@ yarn add @workday/canvas-kit-labs-react-pagination
 
 ## Usage
 
+`Pagination` is a composable component, meaning it can be put together and used in a variety of
+different ways. We'll first cover basic usage examples followed by advanced usage. Each of the
+examples is intended to be standalone, so you can skip to the section that best fits your use case.
+
+These examples are designed to provide copy-and-paste snippets to help you get up and running
+quickly without much explanation. Further detail on individual component props, behavior, and
+accessibility can be found in the [Components](#components) section.
+
+### Basic Usage
+
+Below are examples we expect will work for most use cases. Please refer to the Advanced Usage
+section for details on supporting custom implementations.
+
+#### Pagination with Step Controls
+
+In this example, the component will only display the step controls (next page and previous page) and
+the page list buttons.
+
+```tsx
+import * as React from 'react';
+import Pagination from '@workday/canvas-kit-labs-react-pagination';
```

Comment:
The Pagination imports will need to be updated to `import {Pagination}` if there's no longer a default export.

Reply:
Great catch!